shell/platform/android/io/flutter/embedding/android/FlutterSurfaceView.java

```diff
@@ -91,6 +92,10 @@ private void init() {
     // can monitor changes and forward those changes on to native Flutter code.
     getHolder().addCallback(surfaceCallback);
 
+    // Set this SurfaceView's PixelFormat to TRANSPARENT so that there is no initial black rectangle
+    // visible for a brief period of time before the alpha is set to zero.
+    getHolder().setFormat(PixelFormat.TRANSPARENT);
```

Comment:
On the iOS side, this is optional and I believe we default it to false (both because of performance reasons and to not break someone who relied on this behavior in the past).  Can we do the same here?

Reply:
e.g. make it a constructor parameter or a property on the relevant class(es) exposed to the app

Reply:
For now, by keeping this an implementation detail, we are free to find a better solution without changing the public API. If this is really the only answer then we can expose a public API once we arrive at that conclusion. But I'm hoping we can find another way around this issue.

Reply:
Wouldn't you want to use this if you had a non-rectangular widget to show over your native view?

Reply:
I'm not sure I understand the question. This is addressing a black flicker issue that appears to occur no matter what you're showing in the Flutter UI.

Reply:
(Discussed offline - this is good for now)